src/OrchardCore.Modules/OrchardCore.ContentTypes/AdminMenu.cs

```diff
@@ -40,12 +40,12 @@ protected override ValueTask BuildAsync(NavigationBuilder builder)
         builder
             .Add(S["Design"], content => content
                 .Add(S["Content definition"], S["Content definition"].PrefixPosition(), contentDefinition => contentDefinition
-                    .Add(S["Content types"], S["Content types"].PrefixPosition("1"), contentTypes => contentTypes
+                    .Add(S["Types"], S["Types"].PrefixPosition("1"), types => types
```

Comment:
I'd keep this and "Content parts". Yes, it's more verbose, but that doesn't matter and it's also clearer, because "type" and "part" are overloaded terms.

Reply:
I actually removed all the duplicates terms.  In this case, the parent menu is Content definition. So type and parts is much simpler and cleaner than having to repeat context again "Content Types" and "Content Parts".

Reply:
I don't think cleanliness or simplicity is a priority over clarity here.

Reply:
I think it's clear. But no strong opinion. If you have strong opinion on being repeating context, I'll prepend the word "Content".

Reply:
My opinion is strong on keeping these links :). It's not really repeating: you and I and everybody in this PR understand this, but what is listed under e.g. Content definition -> Types, are "content types", not "content definition types", or simply "types".

OC users have to understand and follow a bunch of terminology, let's make this easier whenever possible. (And as a bonus, not have yet another new translation string.)

Reply:
done

Reply:
Thanks!